typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -36,7 +32,7 @@ export async function placeLimitOrderExample() {
   const marketAddress = new PublicKey(
     "CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N"
   );
-  const market = phoenixClient.markets.get(marketAddress.toBase58());
+  const market = phoenixClient.marketStates.get(marketAddress.toBase58());
```

Comment:
I know this is super annoying, but here and everywhere else we should stop using the word `market` and call things `marketState`, `marketMetadata`, etc